### PR TITLE
add 0xcb helios

### PR DESCRIPTION
**Pull request by Conor-Burns**
Adding 0xCB Helios

[source](https://github.com/0xCB-dev/0xCB-Helios)

### CODE REVIEW

File: src/boards/include/boards/0xcb_helios.h [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+// -----------------------------------------------------
+//
+//------------------------------------------------------------------------------------------
+// Board definition for the 0xCB Helios
+
+#ifndef _BOARDS_0XCB_HELIOS_H
+#define _BOARDS_0XCB_HELIOS_H
+
+// For board detection
+#define 0XCB_HELIOS
```
Comment (kilograham) [inline src/boards/include/boards/0xcb_helios.h:19]: Um; that surely doesn't compile!

**Review by kilograham — CHANGES_REQUESTED** [review]
as per comment, this header won't compile

**Comment by kilograham** [comment]
please fix this if you want it

**Comment by kilograham** [comment]
As OP doesn't seem to be around, I suggest we just add an `_` to the define and commit this (i would have done so in passing, but i can't edit the commit in the original repo)

**Comment by lurch** [comment]
> As OP doesn't seem to be around, I suggest we just add an `_` to the define and commit this (i would have done so in passing, but i can't edit the commit in the original repo)

Done that in #1724

**Comment by kilograham** [comment]
#1724 has been merged into develop

**Pull request closed without merging**